src/modules/pulseaudio.cpp

```diff
@@ -26,6 +26,7 @@ namespace modules {
 
     auto sink_name = m_conf.get(name(), "sink", ""s);
     bool m_max_volume = m_conf.get(name(), "use-ui-max", true);
+    m_invert_scroll = m_conf.get(name(), "invert-scroll", false);
```

Comment:
Please call the option `reverse-scroll` to be consistent with other modules that also have such an option.